cookbook/docs/contribute.rst

```diff
@@ -79,8 +84,9 @@ Below you’ll find instructions that will hopefully guide you through how to co
    .. tip::
       For implicit targets, run ``make -C docs html``.
    4. Open HTML pages present in the ``docs/_build`` directory in the browser
-   
-5. After creating the pull request, check if the docs are rendered correctly by clicking on the documentation check 
+
+
+5. After creating the pull request, check if the docs are rendered correctly by clicking on the documentation check.
    
    .. image:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flytesnacks/contribution-guide/test_docs_link.png
        :alt: Docs link in a PR
```

Comment:
In the end, add:

You can refer to [this PR](https://github.com/flyteorg/flytesnacks/pull/332) for the exact changes required.